DiverseSelector/diversity.py

```diff
@@ -42,7 +42,8 @@
 
 def compute_diversity(
     features: np.array,
-    div_type: str = "total_diversity_volume",
+    div_type: str = "entropy",
+    library: np.array = None,
```

Comment:
The wording of `library` is too generic and can be confusing as we often use it to denote software packages. Maybe change it to `features` or something that's clear?